lib/models/device_info.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:collection';
+
+class SupportDeviceInfo extends UnmodifiableMapView<String, String?> {
```

Comment:
Using a hardcoded map like this makes it hard to localize.
I'd rather keep it as a simple const data class, and render it into a map where it's being used.
Would be awesome if you localized the keys too.

Reply:
Added the code for map creation as part of the class because the map is used in three different widgets. Otherwise, need to create a utility function.